docs/GoogleDrive-batchsource.md

```diff
@@ -63,27 +63,22 @@ OAuth2 client credentials can be generated on Google Cloud
 
 For more details on OAuth2, see [Google Drive Api Documentation](https://developers.google.com/drive/api/v3/about-auth)
 
-#### Service Account Properties
+#### Service Account Type
 
-**Account File Path:** Path on the local file system of the service account key used for authorization. 
-Can be set to 'auto-detect' for getting service account from system variable.
-The file/system variable must be present on every node in the cluster.
-Service account json can be generated on Google Cloud 
-[Service Account page](https://console.cloud.google.com/iam-admin/serviceaccounts)
+Make sure that the Google Drive Folder is shared to the service
```

Comment:
Can you you say "is shared with" instead of to (this matches the google drive sink text.  Also, do you need info about Private Google Drives and Shared Google Drives or do they all work with the Viewer role?